### ISSUE TITLE
Don't report multiple h1 tags as mistake

### ISSUE THREAD

**Issue by pgalle** [Elderjs/elderjs#184]
It's a long debate if people should use one or multiple h1 tags if they fit.

A while ago it has been answerded [here](https://www.youtube.com/watch?v=zyqJJXWk0gk)

Essentially this video is my main point of reference. Other than that I'm using [NVDA](https://nvda.bhvd.de/) to test my application for accessibility manually and [axe-core](https://github.com/dequelabs/axe-core) within automated frontend tesing. Multiple h1 have never been reported as an issue.

Using NVDA you can see and hear the heading ordered by their hierarchy. It's kinda like calling 
`document.querySelectorAll('h1, h2, h3, h4, h5, h6')` in your console and indenting each h2 below their previous h1, etc.

The main topic of a page is defined by its `title` tag. This is was being read first if you open up a website. Afterwards you're free to do what you want. So people look at landmarks, forms, links, sometimes ids and ofcourse headings.

A good example would be a page like this:

```
title: cake recipes
h1: cherry cake
  h2: ingredients
  h2: how to make the cake
h1: carrot cake
  h2: ingredients
  h2: how to make the cake
h1: banana cake
  h2: ingredients
  h2: how to make the cake
```

If those users hear that there is only one h1 on the page they might get confused when the list is that long. In case of https://elderguide.com/tech/elderjs/ we have something like this:

title: `Elder.js: A Svelte Framework and Static Site Generator` (that's being read on page load)
h1: `Elder.js: An 'SEO-First' Svelte-Based Framework` (the only h1, it's essentially the same as the title)

Afterwards the structure seems correct. 

In short the issue I see is, that the h1 provides no value in this case for screen reader users. It should've been a span. All headings below should be decremented by one. This would reduce the depth of the heading tree which helps users with cognitive difficulties.
What's good for accessibility is good for SEO in the long run.

Kind regards

**Comment by nickreese** [comment]
In my experience multiple H1s never ends well ranking wise. Last time I tested was a few years back and it wasn’t a perfect test but the pages structured against common SEO advice didn’t do well. 

That said, the only context where multiple h1s appears to be semantically correct is when there are multiple `section` are on the page. (I would accept a PR that allowed this edge case)  

This plugin is by design opinionated. 

SEO is a guessing game and it is impossible to draw causation these days unless you’re doing A/B split testing at the template level with pages that have similar link equity, traffic, and keyword difficulty/competition. We did this ok broadbandnow.com and used Facebook’s Prophet package to estimate the impact of changes. We never test multiple H1s but we tested a lot of other things. 

The resources and know how needed to run this type of split test isn’t common in the industry, yet you’ll see tons of people spouting their opinions on what works best based on their assumptions. 

I regularly disagree with people about best practices and I’m OK with that. I will trust years of experience over an article or what the current Google spokesperson says. 

I think the best path forward is a PR that allows people to define their own rules.

**Comment by nickreese** [comment]
@pgalle Created a way to add your own rules in the latest release of SEO check. Also exported the default rules too. 

https://github.com/Elderjs/plugins/tree/master/packages/seo-check
